crypto/x509/x509_vfy.c

```diff
@@ -986,15 +1011,142 @@ static int check_revocation(X509_STORE_CTX *ctx)
     }
     for (i = 0; i <= last; i++) {
         ctx->error_depth = i;
-        ok = check_cert(ctx);
-        if (!ok)
-            return ok;
+#ifndef OPENSSL_NO_OCSP
+        if (ctx->param->flags & X509_V_FLAG_OCSP_RESP_CHECK) {
+            ok = check_cert_ocsp_resp(ctx);
+
+            /*
+             * If the client receives a "ocsp_response_list" that does not
+             * contain a response for one or more of the certificates in the
+             * completed certificate chain, the client SHOULD attempt to
+             * validate the certificate using an alternative retrieval method,
+             * such as downloading the relevant CRL;
```

Comment:
This first half of the comment fits better just before the `#endif` currently in line 1047,
i.e., after trying to use OCSP and before the original checks using CRLs.

Reply:
Done

Reply:
It does not seem to be pushed/committed?

Reply:
Is committed and pushed